docs/inso-cli/cli-command-reference/inso-export-spec.md

```diff
@@ -21,6 +21,8 @@ inso export spec [identifier]
 Option  | Alias | Description
 ----- | ------ | -------
 `--output <path>` | -o | Save the specification to a file in the working directory.
+`--workingDir <path>` || Specifying the current working directory.
```

Comment:
```suggestion
`--workingDir <path>` || Specify the current working directory.
```
Is there a default? Is the default just `$HOME`? I'd add that to the description.

Reply:
Oh I agree that wording is confusing.  Without --workingDir, the default is to use the current folder the program is running in as the "working directory".  When you set --workingDir <path>, then the program will look at the folder in <path> and use that as the working directory.

I guess the analogy is:  I tell you to cook an egg in the kitchen.  You'll, by default, think the kitchen is the one where you are now.  If I say "Cook an egg in the kitchen in my house", now you know to not cook an egg wherever you are now, but to come to my house and my kitchen and cook it.

Did that make sense?